src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/doclets.properties

```diff
@@ -342,4 +342,33 @@ doclet.search.packages=Packages
 doclet.search.classes_and_interfaces=Classes and Interfaces
 doclet.search.types=Types
 doclet.search.members=Members
-doclet.search.search_tags=Search Tags
\ No newline at end of file
+doclet.search.search_tags=Search Tags
+
+doclet.snippet.contents.none=\
+ @snippet does not specify contents
+
+doclet.snippet.contents.ambiguity.external=\
+ @snippet specifies multiple external contents, which is ambiguous
+
+doclet.snippet.region.not_found=\
+ region not found: "{0}"
+
+doclet.tag.attribute.value.illegal=\
+ illegal value for attribute "{0}": "{1}"
+
+doclet.tag.attribute.repeated=\
+ repeated attribute: "{0}"
```

Comment:
check the resource file for other quoted values; I think you'll find single quotes are the prevailing standard.

Reply:
I didn't find single quotes to be the prevailing standard neither in `doclet.properties` nor in `compiler.properties`. In fact, it seems to be the opposite: whenever a parameter (not constant) is output, it's either unquoted or double-quoted. Did I look in the right place?

Reply:
OK, it's not as dominant as I thought ... 2 to 1.
Grepping through English properties files `jdk.compiler` and `jdk.javadoc`, there are 142 lines containing `'` and 75 lines containing `"`
Maybe we should have a separate cleanup, after figuring out which form we should use.